core/assets/bundles/bundle_vi.properties

```diff
@@ -705,24 +705,24 @@ planet.serpulo.name = Serpulo
 planet.erekir.name = Erekir
 planet.sun.name = Mặt trời
 
-sector.impact0078.name = Impact 0078
+sector.impact0078.name = Tổ hợp Impact 0078
 sector.groundZero.name = Ground Zero
-sector.craters.name = The Craters
-sector.frozenForest.name = Frozen Forest
-sector.ruinousShores.name = Ruinous Shores
-sector.stainedMountains.name = Stained Mountains
-sector.desolateRift.name = Desolate Rift
-sector.nuclearComplex.name = Nuclear Production Complex
+sector.craters.name = Miệng núi lửa
+sector.frozenForest.name = Rừng tuyết trắng
+sector.ruinousShores.name = Bờ biển hoang tàn
+sector.stainedMountains.name = Dãy núi xám màu
+sector.desolateRift.name = Khe núi Tử Địa
+sector.nuclearComplex.name = Khu tổ hợp Sản xuất Hạt nhân
 sector.overgrowth.name = Overgrowth
-sector.tarFields.name = Tar Fields
-sector.saltFlats.name = Salt Flats
-sector.fungalPass.name = Fungal Pass
-sector.biomassFacility.name = Biomass Synthesis Facility
-sector.windsweptIslands.name = Windswept Islands
-sector.extractionOutpost.name = Extraction Outpost
-sector.planetaryTerminal.name = Planetary Launch Terminal
-sector.coastline.name = Coastline
-sector.navalFortress.name = Naval Fortress
+sector.tarFields.name = Thung lũng dầu
+sector.saltFlats.name = Đồng bằng muối
+sector.fungalPass.name = Khe Nấm Bào Tử
+sector.biomassFacility.name = Cơ sở tổng hợp sinh khối
+sector.windsweptIslands.name = Quần đảo Gió Cuốn
+sector.extractionOutpost.name = Tiền đồn chế xuất
+sector.planetaryTerminal.name = Khu bệ phóng Liên Hành Tinh
+sector.coastline.name = Bờ biển
+sector.navalFortress.name = Pháo đài Hải quân
```

Comment:
You shouldnt translate sector name because its make player in other country know what are you playing... or searching map infomation.